### ISSUE TITLE
Can't build anything in game

### ISSUE THREAD

**Issue by Chaos192** [Dyonisian/3rdRPG#1]
I compiled it for 4.21 and the shooting works no compiler issues but I can't figure out how to do building. I looked at input and it's Z, X, C, V keys but none of them work. How to set it up in blueprint to make it build using the keys like in your video?

**Comment by Dyonisian** [comment]
Hello! Did you also download and add the extra required assets? 

When you press the ZXCV keys, do you see a transparent blue tile appear (Like in the video) to show you can build there? If so, you're supposed to left click to build. Let me know if even the translucent tiles don't appear.

Are there any errors? I remember there's an issue with the character blueprint - I have to re-assign the building blueprints everytime I make a change to the character blueprint. That might be the problem. 

Also, I'm just getting back to this project, probably tomorrow, so I'll check it once more and let you know.

How'd you find this project btw? Genuinely interested in knowing :)

**Comment by Chaos192** [comment]
Yeah I got all the assets from the marketplace. When I press the keys nothing shows at all and in the character BP it's not assigned anywhere for building. 

I did have one deprecated function that prevents it to be upgraded to 4.23 but I left that alone and kept it at 4.21.2 as it built with no errors. In editor there are no map errors or any BP errors. Just nothing is showing.

I found this project when I was searching fortnite on github. I also found another one that is multiplayer but your video I liked the look/feel more than the other one I found.
